integration-tests/hardhat.config.ts

```diff
@@ -10,7 +10,7 @@ const enableGasReport = !!process.env.ENABLE_GAS_REPORT
 
 const config: HardhatUserConfig = {
   mocha: {
-    timeout: 20000,
+    timeout: 150000,
```

Comment:
iirc this was lowered on purpose previously so that tests that would timeout anyways would timeout faster in CI. `ocha` can be passed a `--timeout` flag so we could take advantage of that in CI if we wanted to

Reply:
I'll check the hardhat docs to see if this is configurable when running `npx hardhat test`

Reply:
Not easily configurable so I'm going to leave it at this for now.

Reply:
How often will we run against production networks vs how often do we run integration tests in CI/locally? IMO it'd be easier to keep the number low, and temporarily modify when testing against production.

Reply:
Or even better, it looks like you can set a timeout per network according to [this comment](https://discord.com/channels/750408878008827925/821025292775718962/857631406985576488) in the Hardhat discord: 

> You can configure a timeout for your tests by setting `mocha.timeout` or for a network (e.g. localhost) by setting `networks.localhost.timeout`

Reply:
Perfect. Also added a separate `yarn test:integration:live` command to simplify this further.